tools/unicode_c_source.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+LICENSE = """/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */"""
+
+
+class Source(object):
+    def __init__(self, filepath):
+        self.__filepath = filepath
+        self.__header = LICENSE
+        self.__data = ""
+
+    def complete_header(self, completion):
+        header = self.__header[:len(self.__header) - 3]
+        header += " *\n" + completion
+        header += " */\n\n"
+        self.__header = header
+
+    def add_table(self, table, table_name, table_type, table_descr):
+        self.__data += (table_descr + "\n"
+                        "static const " + table_type + " jerry_" + table_name + "[] JERRY_CONST_DATA =\n"
+                        "{\n" + format_code(table, 1) + "\n};\n\n")
```

Comment:
If the `__data` is a list then this could look like this:
```
self.__data.append(table_descr)
self.__data.append("static const %s jerry_%s[] JERRY_CONST_DATA" % (table_type, table_name))
self.__data.append("{")
self.__data.append(format_code(table, 1))
self.__data.append("}")
self.__data.append("") # for an extra empty line
```
then when writing out the data we can just simple join them together. What do you think?